dlp/src/test/java/dlp/snippets/InspectTests.java

```diff
@@ -124,60 +122,30 @@ public void testInspectImageFile() {
   }
 
   @Test
-  public void testInspectGcsFile() throws InterruptedException, ExecutionException, IOException {
-
+  public void testInspectGcsFile() throws Exception {
```

Comment:
It's actually best to throw the most specific exceptions so that users understand and can mitigate what's going on.  That said, in a test, this is less important.

Reply:
See below.